modules/claims_api/lib/bgs_service/power_of_attorney_request_service.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class PowerOfAttorneyRequestService < ClaimsApi::LocalBGS
+    PROC_TYPE_CD = 'POAAUTHZ'
+
+    def proc_create
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        <arg0>
```

Comment:
namespaces do not seem to matter for `vnpProcCreate` which is something we'll have to solve for in the future

Reply:
interesting. well when we need them, that'll be unfortunate for anyone working that ticket 😢